examples/remix/backend/buckets.server.ts

```diff
@@ -0,0 +1,13 @@
+import { S3Client, ListBucketsCommandInput, ListBucketsCommand, CreateBucketCommandInput, CreateBucketCommand } from '@aws-sdk/client-s3';
+
+export async function listBuckets(s3: S3Client, bucketsParams: ListBucketsCommandInput) {
+    const data = await s3.send(new ListBucketsCommand(bucketsParams));
```

Comment:
It seems like we are always using the `send` method when interacting with AWS s3 client, and the command would be the argument inside `send` which takes an object of params as argument?